### PR TITLE
Issue #1259

### PR DESCRIPTION
**Pull request by divyanshumehta**
Solves the Issue #1259 . Counts currently ongoing conference and redirects to the conference#show view if there is only one live conference

### CODE REVIEW

**Comment by differentreality** [comment]
@divyanshumehta can you please add in the comments a description of what your PR is doing and which issue it is solving?

File: app/controllers/conferences_controller.rb [hunk 1]
```diff
@@ -6,6 +6,9 @@ class ConferencesController < ApplicationController
 
   def index
     @current = Conference.where('end_date >= ?', Date.current).reorder(start_date: :asc)
+    if @current.count == 1
+      redirect_to "/conferences/#{@current.first.id}"
```
Comment (shlok007) [inline app/controllers/conferences_controller.rb]: Did you test the changes in your local machine first? This won't work as routing for conferences is by `short_title` and not `id` .
Also, I suggest using named routes instead. :slightly_smiling_face:

Comment (divyanshumehta) [inline app/controllers/conferences_controller.rb]: Yes @shlok007 it does not work, as routing is based on short_title, also when I the URL `/conferences/:id` with the short_title in `params[:id]`, it redirects me back to homepage saying to login and even after login in it only works for the admin, any user or non-user cannot acess the conference#show show authorization needs to be relaxed in this case.

Comment (nikhilgupta1211) [inline app/controllers/conferences_controller.rb]: @divyanshumehta  the conference should have a splash page and public access for anyone to visit the link

**Pull request closed without merging**